### PR TITLE
Automatic opening

### LINKED ISSUE #29: Enable automatic diagram preview
**What would you like to be added**:

Enable automatic diagram preview.

The diagram should automatically open when you open the workflow file.

**Why is this needed**: 

If you open any serverless workflow file (*.sw.json, *.sw.yml), you have to open the Command Palette (⇧⌘P or Ctrl + Shift + P), then type and select the option "Serverless Workflow: Preview Diagram" to see the workflow's diagram.

### PR DESCRIPTION
**Pull request by JBBianchi**
**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:
As asked in #29, automatically opens a diagram for JSON/YAML when the filename contains `.sw.`

**Special notes for reviewers**:

**Additional information (if needed):**
During local debugging, I'm facing a strange behavior. I need to first open a diagram manually for the automatic behavior to kick-in. If anybody else can test, that would be great <3

### CODE REVIEW

File: resources/diagram-panel.html [hunk 1]
```diff
@@ -12,24 +12,23 @@
...
-              ...vscode.getState(),
-              ...partialState
-            });
-          }
-          
+          /**
+           * Renders the provided graph definition
+           */
           async function updateDiagram({ graphDefinition }) {
+            debugger
```
Comment (ricardozanini) [inline resources/diagram-panel.html]: Leftovers?

Comment (JBBianchi) [reply resources/diagram-panel.html]: Indeed, good catch

**Review by ricardozanini — APPROVED** [review]

**Review by cdavernas — APPROVED** [review]
LGTM! Cheers!

**Comment by ricardozanini** [comment]
@JBBianchi are we ready?

**Comment by JBBianchi** [comment]
Sure, let's say we'll collect live feedback :D

**Comment by ricardozanini** [comment]
Fixes #29

**Pull request merged**